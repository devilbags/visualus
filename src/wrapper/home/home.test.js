import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import { shallow } from 'enzyme';
import Home from './home';

describe('Home wrapper', () => {
  it('Home page wrapper renders without crashing', () => {
    shallow(<Home />);
  });
  it('Should match shallow snapshot', () => {
    const renderer = new ShallowRenderer();
    const result = renderer.render(<Home />);
    expect(result).toMatchSnapshot();
  });
});
