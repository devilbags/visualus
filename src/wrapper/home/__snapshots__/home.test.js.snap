// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home wrapper Should match shallow snapshot 1`] = `
<Adapter
  style={
    Object {
      "padding": "50px",
    }
  }
>
  <div
    className="home"
  >
    <h1>
      This is home
    </h1>
  </div>
</Adapter>
`;
